packages/calcite-components/src/components/combobox/resources.ts

```diff
@@ -3,15 +3,18 @@ export const ComboboxItemGroupSelector = "CALCITE-COMBOBOX-ITEM-GROUP";
 export const AllComboboxChildrenSelector = `${ComboboxItemSelector}, ${ComboboxItemGroupSelector}`;
 
 export const CSS = {
-  input: "input",
-  inputHidden: "input--hidden",
   chip: "chip",
   chipInvisible: "chip--invisible",
-  selectionDisplayFit: "selection-display-fit",
-  selectionDisplaySingle: "selection-display-single",
-  listContainer: "list-container",
   icon: "icon",
+  input: "input",
+  inputHidden: "input--hidden",
+  label: "label",
+  labelIcon: "label-icon",
+  listContainer: "list-container",
   placeholderIcon: "placeholder-icon",
+  selectAll: "select-all",
+  selectionDisplayFit: "selection-display-fit",
```

Comment:
nitpick: `selection-display--single` and `selection-display--fit` if we're following BEM guidelines.